test/e2etest/test-config.tfvars.tmpl

```diff
@@ -26,7 +26,7 @@ resource_tags = {
 }
 
 # Instance types that is preferred for node group.
-instance_types = ["m5.xlarge"]
+instance_types = ["m5.2xlarge"]
```

Comment:
This does make sense. We need to scale less but we still scale up to test the autoscaler.